libs/sdk-ui-vis-commons/src/utils/calculateCustomHeight.ts

```diff
@@ -1,10 +1,16 @@
 // (C) 2021 GoodData Corporation
-import { IHeadlineDataItem } from "../../Headlines";
 
-// Obtained from substrabting the widget title and paddings/margins of visualization.
-export const MINIMUM_HEIGHT_FOR_PAGINATION = 174;
+//Obtained from substrabting the widget title and paddings/margins of visualization.
+const MINIMUM_HEIGHT_FOR_PAGINATION = 160;
 
-export enum HEIGHT {
+/**
+ * @public
```

Comment:
This should be `@internal` since it is in the `vis-commons` package and only for internal use.